pennylane_orquestra/cli_actions.py

```diff
@@ -0,0 +1,231 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This module contains utilities and auxiliary functions for using the Orquestra
+Quantum Engine command line interface (CLI).
+"""
+import subprocess
+import time
+import os
+import urllib.request
+import json
+import tarfile
+
+import yaml
+from appdirs import user_data_dir
+
+
+def qe_get(workflow_id, option="workflow"):
+    """Function for getting information via an Orquestra Quantum Engine CLI
+    call.
+
+    This function is mostly used for retrieving workflow related information.
+
+    Args:
+        workflow_id (str): the workflow id for which information will be
+            retrieved
+
+    Kwargs:
+        option (str): The option specified for the ``qe get`` CLI call.
+            Examples include ``workflow`` and ``workflowresult``.
+
+    Returns:
+        str: response message of the CLI call
+    """
+    process = subprocess.Popen(
+        ["qe", "get", option, str(workflow_id)], stdout=subprocess.PIPE, universal_newlines=True
+    )
+    return process.stdout.readlines()
```

Comment:
Just wanted to check that this `readlines()` returns a full string and not a list of strings?

Reply:
Great catch, thanks! Updated at all occurrences.